src/brevitas/graph/equalize.py

```diff
@@ -431,18 +577,30 @@ def find_sinks(graph_model: GraphModule, starting_node: Node,
             state.sinks.add(None)
```

Comment:
I think we should define a _UNSUPPORTED_OP = object() at the top and then add that and check with is when it's present, cause we are using None already for too much stuff